services/catarse/app/views/catarse_bootstrap/user_notifier/mailer/balance_transfer_error.html.slim

```diff
@@ -2,12 +2,15 @@
 
 - company_name = CatarseSettings[:company_name]
 
-| Oi #{user.display_name},
+- if user.display_name
+  |Oi, #{user.display_name}!
+- else
+  |Oi!
```

Comment:
**Dúvida**
Não seria possível criar um método único e reutilizá-lo nesses templates de e-mail ao invés de replicar esse código em todos os lugares?

Talvez lá no `app/mailers/notifier.rb` dê pra criar algo tipo:
```ruby
def greetings(user)
  return "Oi!" if user.display_name.blank?
  
  "Oi, #{user.display_name}"
end
```

Talvez já ajudasse até a padronizar esses "Oi", "Olá", nome com negrito, nome sem negrito.

Reply:
Eu tava olhando o PR, mas acho que ficaria complicado adicionar, visto que o nome vem de várias estruturas diferentes (contribution, project, user...). Talvez um partial que teste cada uma delas e daí só faz um include?

Reply:
@grpse Tem que ver se é possível colocar esse método aí no `app/mailers/notifier.rb`. Daí se for possível e der pra reutilizar, como ele receber o user como parâmetro, seria só fazer lá dentro do html:
`= greetings(project.user)`
ou 
`= greetings(contribution.user)`
E etc.